src/styles/Blocks/header.scss

```
@@ -0,0 +1,19 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  min-width: 1024px;
+  padding-right: 50px;
+  background: #fff;

```

create variables to all styles
```suggestion
  background: $c-white;
```